src/components/PostList.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Post from './Post';
+
+const PostList = ({ preparedPosts }) => (
+  <ul className="post-list">
+    {preparedPosts.map(posts => (
```

Comment:
```suggestion
    {preparedPosts.map(post => (
```
Насколько я понимаю - сюда должен попадать один комментарий